lib/kb_Metrics/metricsdb_controller.py

```diff
@@ -515,15 +506,13 @@ def join_app_task_ujs(self, exec_tasks, exec_apps, ujs_jobs):
             u_j_s['creation_time'] = u_j_s.pop('created')
             u_j_s['modification_time'] = u_j_s.pop('updated')
             u_j_s['estcompl'] = j.get('estcompl', None)
-            u_j_s['time_info'] = [u_j_s.get('creation_time'),
-                                  u_j_s.get('modification_time'),
-                                  u_j_s['estcompl']]
 
             if u_j_s.get('authstrat', None) == 'kbaseworkspace':
```

Comment:
do if u_j_s.get('authstrat') == 'kbaseworkspace':

by default u_j_s.get('authstrat') is None